<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoiWidget</class>
 <widget class="QWidget" name="RoiWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>934</width>
    <height>338</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>ROI Widget</string>
  </property>
  <layout class="QVBoxLayout" name="roiModeMainLayout">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="mainLayout">
     <item>
      <layout class="QVBoxLayout" name="roiLayout">
       <item>
        <layout class="QHBoxLayout" name="workspaceLayout">
         <item>
          <widget class="QLabel" name="workspaceLabel">
           <property name="text">
            <string>Run</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="textWorkspace"/>
         </item>
         <item>
          <widget class="QPushButton" name="setWorkspaceButton">
           <property name="text">
            <string>Update</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QRadioButton" name="fgRadioButton">
         <property name="text">
          <string>Region of interest</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="bgRadioButton">
         <property name="text">
          <string>Background</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="transRadioButton">
         <property name="text">
          <string>Transmission</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="plotsLayout">
       <item>
        <widget class="QPushButton" name="homeButton">
         <property name="maximumSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="toolTip">
          <string extracomment="Zoom plots out to show all data"/>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="applyButton">
     <property name="toolTip">
      <string>Apply the currently selected regions of interest to the batch settings</string>
     </property>
     <property name="text">
      <string>Apply to Batch</string>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionUpdateWorkspace">
   <property name="text">
    <string>UpdateWorkspace</string>
   </property>
   <property name="toolTip">
    <string>Update the plots following a change to the input workspace</string>
   </property>
  </action>
  <action name="actionHome">
   <property name="text">
    <string>Home</string>
   </property>
   <property name="toolTip">
    <string>Reset pan and zoom to default level</string>
   </property>
  </action>
  <action name="actionApply">
   <property name="text">
    <string>Apply</string>
   </property>
   <property name="toolTip">
    <string>Apply the currently selected regions of interest to the batch settings</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>setWorkspaceButton</sender>
   <signal>clicked()</signal>
   <receiver>actionUpdateWorkspace</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>877</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>textWorkspace</sender>
   <signal>returnPressed()</signal>
   <receiver>actionUpdateWorkspace</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>433</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>homeButton</sender>
   <signal>clicked()</signal>
   <receiver>actionHome</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>879</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>applyButton</sender>
   <signal>clicked()</signal>
   <receiver>actionApply</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>466</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
